src/test/kotlin/interactor/GetTop10CountriesWithHighTaxOnCarbonatedDrinksTest.kt

```diff
@@ -0,0 +1,77 @@
+package interactor
+import org.junit.jupiter.api.Assertions.*
+import dataSource.CsvDataSource
+import dataSource.utils.CsvParser
+import org.junit.jupiter.api.function.Executable
+import org.junit.jupiter.api.Assertions.*
+import dataSource.FakeDataSource
+import org.junit.jupiter.api.*
+import kotlin.Float
+import kotlin.test.assertFailsWith
+import kotlin.Float as Float1
```

Comment:
Why did you use the alias?

```suggestion
import kotlin.Float
```

Reply:
import by mistake I only see it now